oio/cli/admin/directory.py

```diff
@@ -75,7 +76,7 @@ def take_action(self, parsed_args):
 
         if checked:
             self.log.info("Saving...")
-            mapping.force(timeout=(5.0, 30.0))
+            mapping.force(timeout=urllib3.Timeout(connect=5.0, read=30.0))
```

Comment:
You may want to pass `connection_timeout=5.0, read_timeout=30.0` to avoid importing urllib3.